docs/api/checker.rst

```diff
@@ -1021,4 +1024,4 @@ Always available:
    >>> int(bool(PBar))
    1
    >>> int(PBar.__class__ == type)
-   1
+   0
```

Comment:
I admit that I don't understand this bit.

Reply:
Previously, the doctests were only run on Python 3. The base class of `PBar` was defined using Python 2 syntax:
```python
class MyABC:
   __metaclass__ = abc.ABC
```
which didn't carry over to Python 3 (hence `class PBar(MyABC): pass; type(PBar)` *was* `type`. I fixed the test to always set up metaclasses correctly on both versions of Python, so they both properly report `PBar.__class__` as something other than `type`.

Reply:
Ah!  I saw the old tests using `__metaclass__ = ...` and other Python-2-nisms and assumed previously the tests were run only on 2.7.

Also, didn't [testenv:coverage] use basepython = python2.7?  I'm confused now.

Reply:
Ah, but the coverage environment didn't run doctests. (*Only* the `docs` environment ran doctests, and it was---and still is---3.4.)

Reply:
I'm pretty sure all this `>>> int(foo == bar)` business is for compatibility with Python 2.2, where >>> True printed 1 and doctest couldn't cope.  Can we maybe get rid of the int()s and compare with True/False?

Reply:
Yes, that can be done.

Reply:
Whoops, I don't know how I managed to confuse testenv:coverage with testenv:docs in my mind!